BE/prisma/fixtures/projects/web/8.md

```
@@ -0,0 +1,125 @@
+---
+title: web08-JAMstack
+description: 로그인 없이 바로 시작하는 실시간 협업 코드 에디터, CodeJam
+teamName: JAMstack
+demoUrl: http://49.50.138.93/
+thumbnailUrl: https://github.com/boostcampwm2025/web08-JAMstack/raw/dev/docs/assets/slogan_animation.svg
+participants: ['kindsmell', 'Happysttim', 'inaemin', 'son-hyejun', 'lnxhigh']
+---
+
+<p align="center">
+  <img src="./docs/assets/logo_animation.svg" width="120" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)